### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by Diliaver**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,52 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```
Comment (IvanRamyk) [inline app/main.py:2]: list of skills should be an attribute of the concrete developer (not an attribute of the class)

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (Diliaver) [reply app/main.py:2]: Firstly, you should create `SoftwareEngineer` class, 
which will be a parent for all other classes. 
Its constructor should take one parameter: `name` of the 
engineer. **Also, it should have `skills` attribute - 
a list of engineer's skills, such as `"Python"` or `"JavaScript"`.
`skills` should be initialised with empty list.**

Isn't it in the description or I missunderstand something?

Comment (IvanRamyk) [reply app/main.py:2]: Maybe the description isn't occurate. But it's meant that an engineer (an instance of the class) should have a list of his/her skills

Comment (IvanRamyk) [reply app/main.py:2]: We will fix the description to make it clearer

**Review by IvanRamyk — CHANGES_REQUESTED** [review]
please, fix the problem with attributes.
if you have additional questions ask them on slack

**Review by IvanRamyk — APPROVED** [review]
Good job!

**Pull request closed without merging**